compose/src/main/java/com/pouyaheydari/appupdater/compose/dsl/DSLUtils.kt

```
@@ -0,0 +1,106 @@
+import android.graphics.Typeface
+import com.pouyaheydari.appupdater.compose.ui.models.UpdaterDialogData
+import com.pouyaheydari.appupdater.core.model.Theme
+import com.pouyaheydari.appupdater.directdownload.data.DirectDownloadListItem
+import com.pouyaheydari.appupdater.store.domain.StoreListItem
+import com.pouyaheydari.appupdater.store.domain.stores.AppStore
+
+/**
+ * Mutable builder for [StoreListItem] used in DSL context.
+ */
+class StoreListItemBuilder {
+    lateinit var store: AppStore
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)